docs/other/install-gwms-frontend.md

```diff
@@ -366,20 +383,20 @@ Below is an example mapfile, by default found in
 services mentioned above.
 
 !!! note
-    The `example_of_format` entry as each DN should use this format for security purposes.
+    Previously, the `example_of_format` entry was using scape format for each DN like for example: *GSI "^\/DC\=org\/DC\=doegrids\/OU\=Services\/CN\=personal\-submit\-host2\.mydomain\.edu$" %RED<example_of_format>%ENDCOLOR%* for security purposes. This is no longer needed:
```

Comment:
I don't think we need to talk about previous requirements, let's just get rid of this note entirely